spec/plans/provision.fmf

```diff
@@ -62,6 +78,16 @@ example: |
     description:
         Download (if necessary) and start a new container using
         podman or docker.
+
+        image
+            image to use
+        user
+            user name to be used to log in, ``root`` by default
```

Comment:
It isn't `log in`, is it? IMO `user name to be used` is more correct.

Reply:
Removed, per coming PR https://github.com/teemtee/tmt/pull/2549